onnxruntime/contrib_ops/cpu/transformers/beam_search_impl_t5.h

```diff
@@ -214,6 +230,10 @@ Status BeamSearchT5<T>::Execute(const FeedsFetchesManager& encoder_feeds_fetches
   int iteration_counter = 0;
   std::vector<OrtValue> decoder_feeds;
   int current_length = parameters->sequence_length;
+
+  // TODO(tianleiwu): allocate fetches. use ping-pong buffers for past state.
```

Comment:
I think there are some shared buffer for past in GPT-2. Does T5 have similar optimization?

Reply:
yes, https://github.com/microsoft/onnxruntime/pull/15386/files#diff-ba924327241e2d12b13879b683af88533444b675fac501a60d42bc4528fef6c2R265-R277

Reply:
Then we can remove the TODO comment.